readthedocs/proxito/views/serve.py

```diff
@@ -37,38 +37,38 @@
 
 
 class ServePageRedirect(CDNCacheControlMixin, ServeRedirectMixin, ServeDocsMixin, View):
+    def get(self, request, subproject_slug=None, filename=""):
```

Comment:
Don't we need to update the URLConf if we are changing the parameters it receives?

Reply:
urlconf was never sending those other parameters, they were always using the defaults.